gunicorn/arbiter.py

```diff
@@ -558,6 +558,17 @@ def manage_workers(self):
                                   "value": active_worker_count,
                                   "mtype": "gauge"})
 
+        backlog = sum([
+            sock.get_backlog()
+            for sock in self.LISTENERS
+                if sock.get_backlog() is not None
+        ])
```

Comment:
It's better to avoid calling `sock.get_backlog()` twice somehow.

Reply:
`sum(backlog() or 0 for …)`

or `sum(filter(None, [backlog() for …]))`

Reply:
Thanks for the suggestion, fixed it.